src/classes/java/lang/ThreadLocal.java

```diff
@@ -105,15 +105,13 @@ public void remove(){
     class threadSafeFormatter{
       public static ThreadLocal<SimpleDateFormat> df = ThreadLocal.withInitial(() 
           -> new SimpleDateFormat("yyyy-MM-dd"));
-  }
-  
   public class App  {
       public static void main(String[] args) throws Exception {
           threadSafeFormatter tf = new threadSafeFormatter();
           Thread t1 = new Thread();
           t1.start();
```

Comment:
tf.SimpleDateFormat();

Reply:
Hi @ramawatraksha75  Can you please explain a bit more whats the issue is?

Reply:
Sure

Reply:
As I can see we have created the method for treadSafeFormatter but didn't called the function present  in that class

Reply:
I guess it is not necessary

Reply:
Can you please help me out with that

Reply:
This test is maybe not very strong in that it only checks the existence of the alternative constructor, but it is sufficient to check that the old model class of JPF did not support it and the new one does. So there is no need right now to change it.